data_validation/cli_tools.py

```diff
@@ -922,7 +922,12 @@ def _add_common_arguments(optional_arguments, required_arguments):
     optional_arguments.add_argument(
         "--config-file",
         "-c",
-        help="Store the validation in the YAML Config File Path specified",
+        help="Store the validation config in the YAML File Path specified",
+    )
+    optional_arguments.add_argument(
+        "--config-file-json",
+        "-cj",
+        help="Store the validation config in the JSON File Path specified to be used for application purposes, since we only support generating a JSON config file and all other features are exclusive for YAML config files",
```

Comment:
Here too - I think the lengthy description can be confusing... maybe we stick to "Store the validation config in the JSON File Path specified for application use cases"